vxquery-xtest/pom.xml

```diff
@@ -199,20 +199,6 @@
                 </configuration>
             </plugin>
             <plugin>
-                <groupId>org.apache.rat</groupId>
-                <artifactId>apache-rat-plugin</artifactId>
-                <version>${apache-rat-plugin.version}</version>
-                <configuration>
-                    <excludes>
-                        <exclude>**/ExpectedTestResults/**/*</exclude>
-                        <exclude>**/xqts.txt</exclude>
-                        <exclude>test-suite*/**/*</exclude>
-                    </excludes>
-                    <numUnapprovedLicenses>0</numUnapprovedLicenses>
-                    <maxmemory>2g</maxmemory>
-                </configuration>
-            </plugin>
```

Comment:
The exclude statements were getting out of sync with the top level project. Do you know a way to keep them in-sync. Otherwise we can just remove the rat report check from lower level projects and keep it at the top level.